client/components/common/TestPlanResultsTable/utils.js

```diff
@@ -1,15 +1,42 @@
-export const calculateAssertionsCount = testResult => {
-    const passedAssertionsCount = testResult.scenarioResults.reduce(
+export const calculateAssertionsCount = (
+    testResult,
+    includeImpactCalculation = true
```

Comment:
This parameter is unused.

Reply:
True. An optimization I was future-proofing for, but turns out that wasn't needed. Removed in https://github.com/w3c/aria-at-app/pull/849/commits/ca03ee257e27ac755e3dee7fc3323fb59ea01325.